core/trino-main/src/test/java/io/trino/operator/output/TestPositionsAppender.java

```diff
@@ -270,6 +273,7 @@ public void testRowWithNestedFields()
     public static Object[][] types()
     {
         return Arrays.stream(TestType.values())
+                .filter(testType -> !testType.equals(TestType.UNKNOWN))
```

Comment:
nit: why to filter it

Reply:
`UnknownType` does not support values other than null and tests are producing non null blocks based on the type provided

Reply:
would be good to add comment